### DISCUSSION TITLE
[9.x] Illuminate\Session\SymfonySessionDecorator::put()

### DISCUSSION THREAD

**Discussion by afrizal423** [laravel/framework#40533] [Q&A] [answered]
I tried livewire in laravel 9 and I found an error like this
**Call to undefined method Illuminate\Session\SymfonySessionDecorator::put()**

see detail
https://flareapp.io/share/VmeXJwy5#F54

is there any solution for this?

**Comment by nunomaduro** [comment, marked as answer]
Found the issue, working on a patch for this.

**Reply by nunomaduro** [reply]
@afrizal423 Can you try with the latest version of Livewire?

**Reply by afrizal423** [reply]
@nunomaduro  I'm already using the latest version of livewire.
here is my composer.lock file https://pastebin.com/UK83eKt6

**Reply by nunomaduro** [reply]
This will fix it: https://github.com/livewire/livewire/pull/4559.

**Reply by afrizal423** [reply]
ok thanks, i will try later
